docs/react-integration.md

```diff
@@ -172,31 +175,38 @@ The following props will be given to your wrapped component:
  - `fetchStatus`: the status of the fetch (`pending`, `loading`, `loaded` or `error`)
  - `lastFetch`: when the last fetch occurred
  - `hasMore`: the fetches being paginated, this property indicates if there are more documents to load
+ - `isFetching`: a boolean that you can use to show that it's in a fetching state, regardless of the state of the `fetchStatus` variable.
```

Comment:
isFetching should only be there if we have a "background sync" attribute somewhere. Maybe based on an attribute of useQuery() or because we've a global backgroundSync attribute. 

In other case, I don't think this attribute should be set.